### PR TITLE
Add delay flag, bump to 0.8

### PR DESCRIPTION
**Pull request by bhoggard**

### CODE REVIEW

**Comment by bhoggard** [comment]
Should this be updated in the README before this is merged?

`Defaults have changed with the current release. See the CHANGELOG for details.`

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ By default, workers will read from all queues. You can also specify custom `queu
             "worker1": {
               "0": { "queues": "highpriority,images" },
               "1": { "queues": "emails" },
-              "2": { "queues": "images" }
+              "2": { "queues": "bidding", delay: 1 }
```
Comment (bhoggard) [inline README.md]: This should be `"delay"`. Let me know about my other README question, and then I'll push a fix.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Specify an alternate `delayed_job` script location by overriding the `delayed_jo
       "delayed_job": { "pool_size": 3, "path_to_script": "script" }
     }
 
-By default, workers will read from all queues. You can also specify custom `queues` parameters by adding attributes corresponding to each instance name and worker process. A special `default` pool can be specified to supply configuration for instances that don't otherwise appear in the JSON:
+By default, workers will read from all queues. You can also specify custom `queues` parameters by adding attributes corresponding to each instance name and worker process. You can set a specific value for `--sleep-delay` seconds with a `delay` parameter. A special `default` pool can be specified to supply configuration for instances that don't otherwise appear in the JSON:
```
Comment (joeyAghion) [inline README.md]: Could we call it `sleep_delay` to better match the command line argument? Just "delay" is a little ambiguous in the context of this library.

**Comment by joeyAghion** [comment]
Yup--fine to remove that README warning I think.

**Pull request merged**